### ISSUE TITLE
Can we also have None option for aggregation?

### ISSUE THREAD

**Issue by junwoony** [rusty1s/pytorch_geometric#1136]
## 🚀 Feature

It would be also great if we have an option for not aggregating messages, currently we must choose one of `add`, `mean`, and `max`.

## Motivation

 I want to update edges directly based on the  messages for each pair of nodes.

**Comment by junwoony** [comment]
Or if there is a way to output variables before the aggregation that happens in between "message" and "update" functions

**Comment by rusty1s** [comment]
Yes, you can also customize the `aggregate` function, but I am working on making this part more user-friendly. Stay tuned!

**Comment by tobiasfshr** [comment]
customizable aggregate function would be awesome! There are some approaches that use attention mechanisms for this: https://arxiv.org/pdf/1904.05582.pdf (they call it gather function rather than aggregate function)
